### PR TITLE
Add Training operator LoRA test + new fms image

### PR DESCRIPTION
**Pull request by sutaakar**
Tested locally.

### CODE REVIEW

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-tuning-stack-tests.robot [hunk 1]
```diff
@@ -9,13 +9,21 @@ Resource          ../../../tests/Resources/Page/DistributedWorkloads/Distributed
...
-    Run Training Operator ODH Core Test    TestPytorchjobWithSFTtrainer
+    Run Training Operator ODH Core Test    TestPytorchjobWithSFTtrainerFinetuning
+
+Run Training operator ODH test base LoRA use case
+    [Documentation]    Run Go ODH tests for Training operator base LoRA use case
+    [Tags]  RHOAIENG-6965
+    ...     Tier1
+    ...     DistributedWorkloads
+    ...     TrainingOperator
+    Run Training Operator ODH Core Test    TestPytorchjobWithSFTtrainerFinetuning
```
Comment (ChughShilpa) [inline ods_ci/tests/Tests/650__distributed_workloads/test-run-tuning-stack-tests.robot:26]: You are executing the same test `TestPytorchjobWithSFTtrainerFinetuning` here as well

Comment (ChughShilpa) [reply ods_ci/tests/Tests/650__distributed_workloads/test-run-tuning-stack-tests.robot:26]: Isn't it suppose to run ` TestPytorchjobWithSFTtrainerLoRa` test ?

Comment (sutaakar) [reply ods_ci/tests/Tests/650__distributed_workloads/test-run-tuning-stack-tests.robot:26]: Right, thanks

Comment (sutaakar) [reply ods_ci/tests/Tests/650__distributed_workloads/test-run-tuning-stack-tests.robot]: Fixed, can you please check again?

Comment (ChughShilpa) [reply ods_ci/tests/Tests/650__distributed_workloads/test-run-tuning-stack-tests.robot]: lgtm

**Review by ChughShilpa — APPROVED** [review]

**Review by Srihari1192 — APPROVED** [review]

**Review by abhijeet-dhumal — APPROVED** [review]

**Pull request merged**